test/features/wellknown_jwks.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Endpoint to retrieve all public keys of the IdP server. Returns the public key set in JWKS format
+
+  Request endpoint: GET /.well-known/jwks.json
+
+  Scenario: Successfully retrieves the IdP server's public keys
+    When The request to retrieve all public keys of the IdP server is sent
```

Comment:
Here the same what in https://github.com/GovStackWorkingGroup/bb-identity/pull/19#discussion_r1089080030
To discuss :)